install/source/chapter07.rst

```diff
@@ -2,46 +2,36 @@
 7 Setup Worker Node for SSL
 #############################
 
-Login to the Web Portal as the master administrator and go to the "Cluster Manager" page. 
+Login to the Web Portal as the master administrator, then, under the menu next to dashboard, go to "Cluster Control Panel".
 
-.. image:: _static/image_s7_1_1.png
+.. image:: _static/image_s7_1_1_v2.png
 
-Click on the "Cluster Server Farm" page to see the "Cluster Worker Nodes".
+After this, go to "Worker Nodes".
 
-.. image:: _static/image_s7_1_2.png
+.. image:: _static/image_s7_1_2_v2.png
 
 There is an "Advanced Setting" icon on the page.
 
-.. image:: _static/image_s7_1_3.png
+.. image:: _static/image_s7_1_3_v2.png
 
 Now, check **"Always force SSL on Login"** and **"Always force SSL for Native Clients"** boxes.
 
 .. warning::
 
   If you use the self-signed SSL certificate, the web portal is the only client that allows you to login after some SSL certificate warning. All other native clients such as Windows, mobile, and Mac clients will reject the connection.
 
-.. image:: _static/image_s7_1_4.png
+.. image:: _static/image_s7_1_4_v2.png
 
 You will also modify the node property for the node.
 
 The External URL will need to match the node's DNS name and the HTTP or HTTPS protocol.
```

Comment:
This seems wrong, change it to:
The Internal URL will need to match the internal IP address or node's private DNS name and the HTTP or HTTPS protocol.